pkg/golinters/wrapcheck.go

```diff
@@ -1,19 +1,29 @@
 package golinters
 
 import (
-	"github.com/tomarrell/wrapcheck/wrapcheck"
+	"github.com/tomarrell/wrapcheck/v2/wrapcheck"
 	"golang.org/x/tools/go/analysis"
 
+	"github.com/golangci/golangci-lint/pkg/config"
 	"github.com/golangci/golangci-lint/pkg/golinters/goanalysis"
 )
 
 const wrapcheckName = "wrapcheck"
 
-func NewWrapcheck() *goanalysis.Linter {
+func NewWrapcheck(cfg *config.WrapcheckSettings) *goanalysis.Linter {
+	c := wrapcheck.NewDefaultConfig()
```

Comment:
Maybe you can use `settings` instead of `cfg` and `cfg` instead of `c`:
```go
func NewWrapcheck(settings *config.WrapcheckSettings) *goanalysis.Linter {
	cfg := wrapcheck.NewDefaultConfig()
```